### ISSUE TITLE
And another string :( When expecting a time.

### ISSUE THREAD

**Issue by angerman** [mpickering/slack-api#9]
Just invited a bot into a channel.

```json
{"type":"channel_created","channel":{"id":"...","is_channel":true,"name":"...","created":"1423208534","creator":"..."},"event_ts":"1423208533.825935"}
```
when expecting a Time, encountered String instead
Please report this failure to the github issue tracker

looks like slackapi is a little buggy?

**Comment by mpickering** [comment]
Yes.. you would be right in assuming that. I also found it hard to write tests to test every entry point, if you have any suggestions then please let me know.

**Comment by angerman** [comment]
I think it would only help if one could install a validator on slack's end. Otherwise, we'd have to somehow come up with an approach that would make the slack api provide all variations of the json responses in a structured way. We can probably just keep reporting issues to them.
